### ISSUE TITLE
How to build Detox 8.0 on Android

### ISSUE THREAD

**Issue by 2ducanhpham** [wix/detox#857]
#### Error
 What went wrong:
Could not determine the dependencies of task ':app:compileDebugAndroidTestJavaWithJavac'.
> Could not resolve all task dependencies for configuration ':app:debugAndroidTestCompileClasspath'.
   > Could not resolve project :detox.
     Required by:
         project :app
      > Project :app declares a dependency from configuration 'androidTestCompile' to configuration 'newOkhttpDebug' which is not declared in the descriptor for project :detox.

#### Detox, Node, Device, Xcode and macOS Versions

* Detox: 8.0.0
* React Native: 0.55.4
* Node: v10.6.0
* Device: emulator
* Xcode: 9.4.1
* macOS: 10.13.5

**Comment by noomorph** [comment]
Not enough information to start talking about this issue.
Looks like somethings wrong in your Gradle configuration, not Detox itself.

**Comment by 2ducanhpham** [comment]
My build.gradle

androidTestCompile(project(path: ":detox", configuration: "newOkhttpDebug"), {
      exclude group: 'com.android.support', module: 'support-annotations'
    })

**Comment by noomorph** [comment]
Are you sure you have Gradle 3 and followed this article? 
https://github.com/wix/detox/blob/master/docs/Introduction.Android.md

**Comment by 2ducanhpham** [comment]
Yeah ! I have upgraded gradle plugin 3.0.1, but I have not reinstalled detox yet. 
I will try now!
Thanks @noomorph

**Comment by 2ducanhpham** [comment]
Run success ! thank @noomorph 🐳
